cli/sawtooth_cli/network_command/compare.py

```diff
@@ -77,31 +79,57 @@ def do_compare_chains(args):
     urls = split_comma_append_args(args.urls)
     users = split_comma_append_args(args.users)
     clients = make_rest_apis(urls, users)
-    chains = get_chain_generators(clients)
+    chains, unreporting = get_chain_generators(clients)
```

Comment:
Would be useful to print the unreporting nodes.

Reply:
They get printed here: https://github.com/hyperledger/sawtooth-core/pull/1282/commits/a2748f2de4a7ae191d720dd0ff8e12bc30df40fb#diff-43dc287f9866e819f9773544270cc762R112

Reply:
Seems like a side-effect, rather than printing all the unreporting that you return at once.  Makes it hard to track down.